apps/dapp/src/safe/sdk/contracts/config.ts

```diff
@@ -0,0 +1,33 @@
+import { SafeVersion } from '@safe-global/safe-core-sdk-types';
+
```

Comment:
Given this code is sourced from the official safe sdk, can you comment with a link to the original source?